### PR TITLE
Add CODEOWNERS for Page Builder

### PR DESCRIPTION
**Pull request by davemacaulay**
## Description

Add CODEOWNERS for Page Builder

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+/packages/venia-ui/lib/components/RichContent/PageBuilder/ @davemacaulay
```
Comment (jimbo) [inline .github/CODEOWNERS]:
```suggestion
# create-pwa
/packages/create-pwa/lib @zetlen

# peregrine
/packages/peregrine/lib @jimbo

# pwa-buildpack
/packages/pwa-buildpack/lib @zetlen

# venia-ui
/packages/venia-ui/lib @jimbo
/packages/venia-ui/lib/components/RichContent/PageBuilder/ @davemacaulay

# upward-js
/packages/upward-js/lib @zetlen

# upward-spec
/packages/upward-spec/suite @zetlen
```

**Comment by sirugh** [comment]
Oh, just to make sure, there's no other config/etc code that might fall under your umbrella?

**Comment by davemacaulay** [comment]
@sirugh nope, all our code is within this directory, any other changes we made were just to help our stuff work :)

**Comment by davemacaulay** [comment]
R.I.P @zetlen's & @jimbo's inbox

**Comment by jimbo** [comment]
> R.I.P @zetlen's & @jimbo's inbox

Perhaps. Let's let @zetlen chime in before attempting to merge this.

**Pull request merged**